source/meta.tex

```diff
@@ -1356,6 +1356,11 @@
 \begin{note}
 If a type has padding bits, the condition does not hold;
 otherwise, the condition holds true for integral types.
+If \tcode{std::numeric_limits<float>::is_iec559} is \keyword{true}, the
+condition does not hold because there are multiple object representations for
+some values, such as zero.
+The condition might not hold for \keyword{bool} because
+\keyword{true} or \keyword{false} can have multiple object representations.
```

Comment:
This seems to contradict the first sentence of the note, since `bool` is an integral type.

Reply:
Good find, this actually makes the note in the current draft incorrect. I've reworded it so that `bool` is not included.